setup.py

```diff
@@ -20,5 +16,7 @@
         "future>=0.16.0",
```

Comment:
Note for the future (can't recall if I mentioned) - we should just remove, since this should be Py3 only.